136.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var singleNumber = function (nums) {
+  let arr = nums.sort((a, b) => a - b);
```

Comment:
```suggestion
  const arr = nums.sort((a, b) => a - b);
```
Use `const` instead of let